### PR TITLE
translate to french

### PR DESCRIPTION
**Pull request by daliasawaya**

### CODE REVIEW

File: lab01/index.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
     <body>
-        <h1>Hello World</h1>
+        <h1>Bonjour Mode</h1>
```
Comment (G1avina) [inline lab01/index.html]: Bonjour Mode --> Bonjour Monde

**Review by G1avina — COMMENTED** [review]
Decent but there's a spelling mistake LOL Learn how to write !!!!!!!!!

**Review by G1avina — APPROVED** [review]
LGTM!

**Review by polo1250 — APPROVED** [review]
LGTM

**Pull request merged**